mobile_verifier/src/heartbeats/last_location.rs

```diff
@@ -10,21 +10,22 @@ use super::Heartbeat;
 #[derive(Debug, sqlx::FromRow, Copy, Clone, PartialEq)]
 pub struct LastLocation {
     pub location_validation_timestamp: DateTime<Utc>,
-    pub latest_timestamp: DateTime<Utc>,
+    #[sqlx(rename = "first_timestamp")]
+    pub heartbeat_timestamp: DateTime<Utc>,
```

Comment:
If we're updating the name of the field and the column in the database, can they match?